app/scripts/controllers/HomeCtrl.js

```diff
@@ -1,6 +1,13 @@
 (function () {
-    function HomeCtrl(Room, $uibModal) {
+    function HomeCtrl(Message, Room, $uibModal) {
         this.chatRooms = Room.all;
+        this.currentRoom = null;
+        this.messages = null;
+        
+        this.displayCurrentRoom = function(roomName, roomId) {
```

Comment:
FWIW, I found this function name a bit misleading.. I thought it would display information about the current room, not change the current room (I assume based on you clicking on a particular room)